### PR TITLE
Explicitly set all front matter labels in the source files

### PR DESCRIPTION
**Pull request by jdbaldry**
I am seeing some confusing behavior on the production website where page labels change between builds. To ensure that this is not due to cascading front matter, I am replacing it with explicit label declarations in each file.

If this works as intended, Writers' Toolkit and `doc-validator` will be updated to insist that this required front matter is present.

Ideally each CODEOWNER would check the files that they own to ensure the labels make sense.

### CODE REVIEW

**Review by GrafanaWriter — APPROVED** [review]

File: docs/sources/dashboards/assess-dashboard-usage/index.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ keywords:
   - presence-indicator
   - search
   - sort
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
```
Comment (imatwawana) [inline docs/sources/dashboards/assess-dashboard-usage/index.md:19]:
```suggestion
```

File: docs/sources/dashboards/dashboard-public/index.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 aliases:
   - ../features/dashboard/dashboards/
   - dashboard-manage/
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
```
Comment (imatwawana) [inline docs/sources/dashboards/dashboard-public/index.md:9]: @evictorero  or @juanicabanas - can you confirm? it's only available in Cloud, right? 
```suggestion
    - enterprise
    - oss
```

File: docs/sources/dashboards/create-reports/index.md [hunk 3]
```diff
@@ -14,6 +14,7 @@ labels:
   products:
     - cloud
     - enterprise
+    - oss
```
Comment (imatwawana) [inline docs/sources/dashboards/create-reports/index.md:17]:
```suggestion
```

**Review by imatwawana — APPROVED** [review]
I've left a couple changes I'm sure of and one question. But don't let it block you. I can always check it and update later if need be.

File: docs/sources/alerting/set-up/_index.md [hunk 4]
```diff
@@ -1,11 +1,13 @@
 ---
-menuTitle: Set up
 aliases:
   - unified-alerting/set-up/
+description: How to configure alerting features and integrations
 labels:
   products:
+    - cloud
+    - enterprise
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/_index.md:8]:
```suggestion

```

File: docs/sources/alerting/set-up/configure-alert-state-history/index.md [hunk 5]
```diff
@@ -1,19 +1,17 @@
...
   - alerting
   - set up
   - configure
   - alert state history
-
 labels:
   products:
+    - cloud
     - enterprise
     - oss
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-alert-state-history/index.md:13]:
```suggestion
   
```

File: docs/sources/alerting/set-up/configure-alertmanager/index.md [hunk 6]
```diff
@@ -8,6 +8,11 @@ keywords:
   - set up
   - configure
   - external Alertmanager
+labels:
+  products:
+    - cloud
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-alertmanager/index.md:13]:
```suggestion
    
```

File: docs/sources/alerting/set-up/configure-high-availability/_index.md [hunk 7]
```diff
@@ -9,6 +9,11 @@ keywords:
   - tutorials
   - ha
   - high availability
+labels:
+  products:
+    - cloud
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/configure-high-availability/_index.md:14]:
```suggestion
    
```

File: docs/sources/alerting/set-up/meta-monitoring/_index.md [hunk 8]
```diff
@@ -7,6 +7,11 @@ keywords:
   - grafana
   - alerting
   - meta-monitoring
+labels:
+  products:
+    - cloud
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/meta-monitoring/_index.md:12]:
```suggestion
    
```

File: docs/sources/alerting/set-up/migrating-alerts/_index.md [hunk 9]
```diff
@@ -5,6 +5,11 @@ aliases:
   - unified-alerting/difference-old-new/
   - alerting/migrating-alerts/
 description: Upgrade Grafana alerts
+labels:
+  products:
+    - cloud
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/migrating-alerts/_index.md:10]:
```suggestion
    
```

File: docs/sources/alerting/set-up/migrating-alerts/legacy-alerting-deprecation.md [hunk 10]
```diff
@@ -1,12 +1,17 @@
...
 aliases:
   - alerting/legacy-alerting-deprecation/
 description: Legacy alerting deprecation notice
-weight: 109
 keywords:
   - grafana
   - alerting
+labels:
+  products:
+    - cloud
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/migrating-alerts/legacy-alerting-deprecation.md:10]:
```suggestion
    
```

File: docs/sources/alerting/set-up/migrating-alerts/legacy-alerting/_index.md [hunk 11]
```diff
@@ -5,6 +5,11 @@ aliases:
   - /docs/grafana-cloud/legacy-alerting/
   - alerting/migrating-alerts/legacy-alerting/
 description: Legacy alerting
+labels:
+  products:
+    - cloud
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/migrating-alerts/legacy-alerting/_index.md:10]:
```suggestion
    
```

File: docs/sources/alerting/set-up/migrating-alerts/legacy-alerting/grafana-cloud-alerting/_index.md [hunk 12]
```diff
@@ -4,6 +4,11 @@ aliases:
   - /docs/grafana-cloud/how-do-i/grafana-cloud-alerting/
   - /docs/grafana-cloud/legacy-alerting/grafana-cloud-alerting/
 description: Grafana Cloud Alerting
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/migrating-alerts/legacy-alerting/grafana-cloud-alerting/_index.md:11]:
```suggestion
   
```

File: docs/sources/alerting/set-up/set-up-cloud/_index.md [hunk 13]
```diff
@@ -8,10 +8,12 @@ aliases:
   - /docs/grafana-cloud/alerts/grafana-cloud-alerting/
   - /docs/grafana-cloud/how-do-i/grafana-cloud-alerting/
   - /docs/grafana-cloud/legacy-alerting/grafana-cloud-alerting/
+description: How to configure Alerting for Cloud
 labels:
   products:
     - cloud
-description: How to configure Alerting for Cloud
+    - enterprise
+    - oss
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/set-up-cloud/_index.md:16]:
```suggestion
   
```

File: docs/sources/alerting/set-up/provision-alerting-resources/terraform-provisioning/index.md [hunk 14]
```diff
@@ -8,6 +8,11 @@ keywords:
   - alerting resources
   - provisioning
   - Terraform
+labels:
+  products:
+    - cloud
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/provision-alerting-resources/terraform-provisioning/index.md:13]:
```suggestion
   
```

**Pull request merged**